packages/winpanda/windows.build.ps1

```diff
@@ -0,0 +1,18 @@
+$ErrorActionPreference = "stop"
+
+$LIB_INSTALL_DIR="$PKG_PATH/lib/python3.6/site-packages"
+
+New-Item -ItemType Directory -Path "$LIB_INSTALL_DIR"
+
+Copy-Item -Recurse -Path "C:\pkg\build\extra\src\*" "$env:PKG_PATH\bin\"
+
+echo "pip version: $(pip --version)"
+echo "Install packages from wheel files."
+# Install from local source directories (git checkout or extracted tarballs).
+# Order matters. Some of these packages require Cython or build C extensions
+# for speed-up if Cython is availble.
+echo "Install packages from local source directories."
+$packages = ("requests,pySmartDL,docopt,jinja2")
+foreach ($package in $packages){
+  pip install -v --no-deps --install-option="--prefix=$PKG_PATH" --root=/ /pkg/src/$package

```

Comment:
1) this would install deps on a build system? Will it allow deps to be deployed at the target system on some system place reffered by PYTHONPATH env variable?
2) I can't see where the $PKG_PATH comes from

Reply:
this is adding python deps to the package

Reply:
Could you please give an example of package's directory tree after it's deployed at a target system? On which path those deps will reside?

Reply:
please look at `bouncer-deps`